js/utils/getBitcoinPrice.js

```diff
@@ -13,107 +13,115 @@ module.exports = function(currency, callback){
+    {
+      $.ajax({
+        method: "GET",
+        url: "https://blockchain.info/ticker"
+      })
       .done(function (response)
       {
-        //console.log("bitcoinAverage: " + response['24h_avg']);
-        if($.isNumeric(response['24h_avg'])) {
-          btPrices.push(response['24h_avg']);
+        var blockChainCurrencies = {};
+        for (var currency in response) {
+        blockChainCurrencies[currency] = response[currency]['15m'];
         }
+        btPrices.push(blockChainCurrencies);
       })
       .fail(function (jqXHR, textStatus, errorThrown)
       {
-        //console.log("bitcoinAverage request failed:");
-        //console.log(jqXHR);
-        //console.log(textStatus);
-        //console.log(errorThrown);
+        console.log("Blockchain request failed: ");
+        console.log(jqXHR);
+        console.log(textStatus);
+        console.log(errorThrown);
       })
       .always(function ()
       {
-        callCoindesk();
+        callCoinkite();
       });
+    };
 
-  var callCoindesk = function ()
-  {*/
+    var callCoinkite = function ()
+    {
       $.ajax({
         method: "GET",
-        dataType: "json",
-        url: "https://api.coindesk.com/v1/bpi/currentprice/" + currency + ".json"
+        url: "https://api.coinkite.com/public/rates"
       })
           .done(function (response)
           {
-            //console.log("coinDesk: " + response.bpi[currency]['rate']);
-            if($.isNumeric(response.bpi[currency].rate)) {
-              btPrices.push(response.bpi[currency].rate);
+            var coinKiteCurrencies = {};
+            for (var currency in response.rates.BTC) {
+                coinKiteCurrencies[currency] = response.rates.BTC[currency].rate;
             }
+            btPrices.push(coinKiteCurrencies);
           })
           .fail(function (jqXHR, textStatus, errorThrown)
           {
-            //console.log("coinDesk request failed:");
-            //console.log(jqXHR);
-            //console.log(textStatus);
-            //console.log(errorThrown);
+            console.log("coinKite request failed: ");
+            console.log(jqXHR);
+            console.log(textStatus);
+            console.log(errorThrown);
           })
           .always(function ()
           {
-            callBlockchain();
+            callBitcoinAvg();
           });
-    //};
 
+    };
 
-    var callBlockchain = function ()
+    var callBitcoinAvg = function ()
     {
       $.ajax({
         method: "GET",
-        url: "https://blockchain.info/ticker"
+        url: "https://api.bitcoinaverage.com/all"
       })
           .done(function (response)
           {
-            //console.log("blockChain: " + response[currency]['15m']);
-            if($.isNumeric(response[currency]['15m'])) {
-              btPrices.push(response[currency]['15m']);
+            var bitCoinAvgCurrencies = {};
+            for (var currency in response) {
+                if(response[currency].averages) {
```

Comment:
See comment above on currency. In this case your if statement satisfies the need to make sure the property you're looping over belongs to the response object, so you don't need to change that.